kagenti/ui/Home.py

```diff
@@ -18,11 +18,32 @@
 
 import logging
 import os
+import ssl
 import streamlit as st
 from streamlit_oauth import OAuth2Component
 import jwt
+import httpx
 from lib.constants import ACCESS_TOKEN_STRING, ENABLE_AUTH_STRING, TOKEN_STRING
 
+# Disable SSL verification globally for self-signed certificates (OpenShift uses self-signed certs)
+ssl._create_default_https_context = ssl._create_unverified_context
+
+# Monkey-patch httpx to disable SSL certificate verification for OAuth flows
+# This is required because httpx_oauth uses httpx.AsyncClient internally
+original_client_init = httpx.Client.__init__
+original_async_client_init = httpx.AsyncClient.__init__
+
+def patched_client_init(self, *args, **kwargs):
```

Comment:
Pylint warns about missing function docstrings (C0116). If you keep small helper functions, add concise docstrings explaining purpose, arguments, and why they exist. That said, prefer removing the helpers by removing the monkey-patch if possible (see comments below) . If helpers remain, add explicit typing and docstrings.